orttraining/orttraining/training_ops/cpu/quantization/fake_quant.cc

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "orttraining/training_ops/cpu/quantization/fake_quant.h"
+#include "core/providers/common.h"
+
+namespace onnxruntime {
+namespace contrib {
+
+namespace {
+
+template <typename T>
+void FakeQuantPerTensor(OpKernelContext* ctx, const int64_t num_elements, const T* input_data, T quant_scale,
+                        T quant_zero_point, int64_t quant_min, int64_t quant_max, T* fake_quantized_data,
+                        bool* quantization_mask_data) {
+  auto* tp = ctx->GetOperatorThreadPool();
```

Comment:
I am curious should we use MakeEigenArrayMap for kernel implementation instead of operate with parallel for loop?

Reply:
Was not able to find a good way to use these functions with `MakeEigenArrayMap`. Keeping this logic as is.